main.py

```diff
@@ -179,6 +190,6 @@ def construct_google_map_path(travel_df): #, geocodes_df
     geocodes_df = pd.read_csv("Data/geocodes.csv", index_col=1)
     geocodes_df["Visited"] = False
     beer_count_df = pd.Index(beers_df.index).value_counts().to_frame()
-    beer_count_df = beer_count_df.rename(columns={"brewery_id":"beer_count"})
+    beer_count_df = beer_count_df.rename(columns={"brewery_id": "beer_count"})
```

Comment:
This rename seems to be strange.
Maybe there is a naming issue somewhere earlier in the code

Reply:
Before renaming beer_count_df looks like this:
```
      brewery_id
858           57
1072          49
44            38
```

It shows brewery_id and it should be beer_count, since index IS the brewery_id.
After renaming:
```
       beer_count
858           57
1072          49
44            38
```